usb.cpp

```diff
@@ -1,21 +1,13 @@
-/*
- * Copyright  Onplick <info@onplick.com> - All Rights Reserved
- * Unauthorized copying of this file, via any medium is strictly prohibited
- * Proprietary and confidential
- */
```

Comment:
Do you think it would be best to keep this information?

Reply:
This file is a part of the open-source package(MuditaOS) released under license specified by Mudita so I assume it could not stay that way. Apart from that, `usb_stack` repository is public, hence it's even worse that way as we are violating assumptions from this header. It definitely needs clarification.